tests/unit/test_install_scripts.py

```
@@ -391,6 +391,135 @@ def test_install_uses_no_cache_dir(self):
+        fn_lines = []
+        for line in lines[fn_start_idx:]:
+            fn_lines.append(line)
+            depth += line.count("{") - line.count("}")
+            if depth <= 0 and fn_lines:
+                break
+        fn_body = "\n".join(fn_lines)
+
+        # uv must be installed via the venv's pip.
+        assert (
+            "-m pip install" in fn_body and " uv" in fn_body
+        ), "create_venv() must install uv into the venv via 'pip install uv' (JTN-605)"
```

Ruff (0.15.9)

[warning] Assertion should be broken down into multiple parts

(PT018)